data_quality_analysis.ipynb

```diff
@@ -0,0 +1,533 @@
+{
```

Comment:
<p>One of the best anylysis that i have ever seen. :))</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HoretskaKris/Apartment_Prices_in_KR-PL-/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>